dhis-2/dhis-api/src/main/java/org/hisp/dhis/icon/DefaultIcon.java

```diff
@@ -27,515 +27,454 @@
-  WEIGHT("weight", "", new String[] {"weight"}),
-  WOLD_CARE(
-      "wold_care", "World care", new String[] {"world care", "nature", "earth", "protection"}),
-  WOMAN("woman", "", new String[] {"woman", "female", "girl"}),
-  YES("yes", "", new String[] {"yes"}),
-  YOUNG_PEOPLE(
-      "young_people", "Young people", new String[] {"young", "juvenile", "teens", "youth"});
-
-  public static final Set<String> VARIANTS = Set.of("positive", "negative", "outline");
+      "chemical",
+      "assay",
+      "analysis",
+      "malaria",
+      "mosquito"),
+  SYRINGE("syringe", "", "syringe", "injectable", "vaccination"),
+  TAC("tac", "Computed axial tomography"),
+  TB("tb", "TB program"),
+  TRANSGENDER("transgender", "", "transgender"),
+  TRAUMATISM("traumatism", "", "patient", "traumatism"),
+  TRAVEL("travel", "", "travel"),
+  TREATED_WATER("treated_water", "", "treated", "infrastructure", "water"),
+  TRIANGLE_LARGE("triangle_large", ""),
+  TRIANGLE_MEDIUM("triangle_medium", ""),
+  TRIANGLE_SMALL("triangle_small", ""),
+  TRUCK_DRIVER("truck_driver", "", "driver", "truck driver", "worker"),
+  UN_PAVE_ROAD("un_pave_road", "", "pave", "road"),
+  UNDERWEIGHT("underweight", "", "symptom", "patient", "diagnosis", "underweight"),
+  VESPA_MOTORCYCLE("vespa_motorcycle", "", "motorcycle", "transport"),
+  VIH_AIDS("vih_aids", "VIH/AIDS", "hematology", "vih", "STI", "aids", "blood"),
+  VIRUS("virus", "Coronavirus", "covid", "19", "corona", "virus"),
+  VOMITING("vomiting", "", "threw-up", "sickness", "ailment", "vomiting"),
+  WAR("war", "Conflict", "battle", "armed", "war", "tank", "conflict"),
+  WASH_HANDS("wash_hands", "Wash hands", "cleaning", "hands", "wash", "water", "soap"),
+  WATER_SANITATION("water_sanitation", "", "sanitation", "water"),
+  WATER_TREATMENT("water_treatment", "", "water treatment", "water"),
+  WEIGHT("weight", "", "weight"),
+  WOLD_CARE("wold_care", "World care", "world care", "nature", "earth", "protection"),
+  WOMAN("woman", "", "woman", "female", "girl"),
+  YES("yes", "", "yes"),
+  YOUNG_PEOPLE("young_people", "Young people", "young", "juvenile", "teens", "youth");
 
   public static final String SUFFIX = "svg";
 
   @Getter private final String key;
-
   @Getter private final String description;
+  @Getter private final Set<String> keywords;
+  @Getter private final List<String> variants;
 
-  private final String[] keywords;
-
-  DefaultIcon(String key, String description, String[] keywords) {
+  DefaultIcon(String key, String description, String... keywords) {
     this.key = key;
     this.description = description;
-    this.keywords = keywords;
+    this.keywords = Set.of(keywords);
+    this.variants =
+        Set.of("positive", "negative", "outline").stream()
+            .map(variant -> "%s_%s".formatted(key, variant))
+            .toList();
   }
 
-  public Set<String> getKeywords() {
-    return Arrays.stream(this.keywords).collect(Collectors.toSet());
+  public List<Icon> createIcons() {
```

Comment:
```suggestion
  public List<Icon> getIconsWithVariants() {
```

or `toIconsWithVariants` 🤷🏻 

Maybe this is only me but `create` reminds me of persisting. Please resolve this if we never use `create` in that regard in our code.